### PR TITLE
Update map

### PR DESCRIPTION
**Pull request by stoe**
As requested by @shajanjp in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4646#issuecomment-289226923

cc/ @hectorsector

map to look like https://services.github.com/on-demand/intro-to-github/

### CODE REVIEW

**Comment by sjdeak** [comment]
good job 👍

**Comment by suhail-n** [comment]
good job :+1:

**Comment by stoe** [comment]
Thanks @shajanjp for merging this, but we were still missing @hectorsector's review.

I personally like that the person creating the PR is the one merging it in the end after all CI test have run and reviews have been approved. It's not a big deal, and I want to thank you once again for taking such good care of this repository.

:octocat: :sparkling_heart:

File: mapsAPI.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          url: 'https://github.com/' + username + '.png?size=20',
+          size: mapSize,
+          scaledSize: scaledSize,
+          anchor: anchor,
+        },
+      });
+    });
+
+  new MarkerClusterer(map, markers, {
+    imagePath: '../images/cluster/m',
```
Comment (stoe) [inline mapsAPI.js]: Have to change the image path to `imagePath: './images/cluster/m',`.

**Pull request merged**